doc/user/layouts/partials/specifying-aws-credentials.html

```diff
@@ -0,0 +1,50 @@
+### Specifying AWS Credentials
+
+AWS credentials can be provided on a per-source basis via the standard AWS credentials provider chain:
+
+1. Statically provided as part of the `WITH` block in source declaration. These credentials will be
+   written to disk in plain text as well as being easily exposed via introspection commands and so
+   this technique is only recommended for experimentation and development.
+2. Environment variables: `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`,
+   [`AWS_WEB_IDENTITY_TOKEN_FILE`][web-identity], and [many more][env-vars].
+3. `credential_process` command in the AWS config file, by default located at `~/.aws/config`.
+4. AWS credentials file and profile files. By default located at `~/.aws/config` and
+   `~/.aws/credentials` , respectively.
+5. The [IAM instance profile][instance-profile] provided by the Instance Metadata Service. Will
+   only work if running on an EC2 instance with an instance profile.
+
+#### AWS credentials `WITH` options
+
+All AWS Credentials `WITH` options are of type `text`.
+
+Static credentials can be configured by the following `WITH` options:
+
+Field | Description
+------|------------
+`access_key_id` | A valid [access key ID][access] for the AWS resource.
+`secret_access_key` | A valid [secret access key][access] for the AWS resource.
+`token` (optional) | The session token associated with the credentials, if the credentials are temporary
+
+Alternatively, you can specify an profile to assume. This named profile must exist within either the `credentials` or `config` file:
+
+Field | Description
+------|------------
+`profile` | An AWS config profile to assume.
+
+The following `WITH` options can be set with either static credentials, a profile, or while depending on the environment for credentials.
```

Comment:
Did you mean `either static credentials, a profile, or alone`?
```suggestion
The following `WITH` options can be set with either static credentials, a profile, or alone depending on the environment for credentials.
```